internal/build/lifecycle_execution.go

```diff
@@ -3,24 +3,31 @@ package build
 import (
 	"context"
 	"fmt"
+	"io"
 	"math/rand"
 	"os"
 	"path/filepath"
 	"strconv"
+	"time"
+
+	"github.com/BurntSushi/toml"
```

Comment:
I couldn't leave a code suggestion as the lines didn't change, but could we group the imports here as well?

Reply:
To be fixed in https://github.com/itsdarshankumar/pack/pull/1

Reply:
done!